tutorials/scripting/signals_concise_example.rst

```diff
@@ -0,0 +1,110 @@
+.. _doc_signals_concise_example:
+
+Signals, concise example
+========================
```

Comment:
I'm open to suggestions, but please let's not bloat this up. In my opinion too many words is bad design and certainly won't help anyone, either. In my mind PHP language's manual pages are superior to Godot's because of their concise examples: https://www.php.net/manual/en/function.array - That could serve as inspiration (but not as strict guide, of course)